scripts/github_last_contribution.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+'''
+This script takes a list of Github users and delivers the timestamp of
+their last contribution.
+'''
+import argparse
+import json
+import requests
+
+from datetime import datetime, time
+
+def last_contribution(username, repo):
+    address = 'https://api.github.com/repos/{0}/{1}'.format(username, repo)
+    request = requests.get (address)
+    if request.ok:
+        user_data = json.loads(request.text or request.content)
+        return user_data['pushed_at']
+    else:
+        print ("Error while fetching data from Github! Status code: " + str(request.status_code))
+
+def get_datetime(date):
+    return None if date is None else datetime.strptime(date, "%Y-%m-%dT%H:%M:%SZ")
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("usernames",
+                        help="Enter usernames separated with spaces.",
+                        type=str,
+                        nargs='+')
+    args = parser.parse_args()
+
+    result = []
+
+    for username in args.usernames:
+        date_zulip = get_datetime(last_contribution(username, 'zulip'))
+        date_zulip_gci = get_datetime(last_contribution(username, 'zulip-gci'))
+
+        if date_zulip is not None and date_zulip_gci is not None:
+            last_date = date_zulip if date_zulip > date_zulip_gci else date_zulip_gci
```

Comment:
would it work with `max` function?
`last_date = max(date_zulip, date_zulip_gci)`